### PR TITLE
OCM-4714 | include --properties in create cluster output

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

File: cmd/create/cluster/cmd.go [hunk 1]
```diff
@@ -3555,6 +3556,9 @@ func buildCommand(spec ocm.Spec, operatorRolesPrefix string,
 			securitygroups.SgKindFlagMap["Control Plane"], strings.Join(spec.AdditionalControlPlaneSecurityGroupIds, ","))
 	}
 
+	if len(properties) > 0 {
+		command += fmt.Sprintf(" --properties %s", strings.Join(properties, ","))
```
Comment (andreadecorte) [inline cmd/create/cluster/cmd.go:3560]: shouldn't this be multiple --properties invocations? I am not sure it works this way

Comment (thomasmckay) [reply cmd/create/cluster/cmd.go:3560]: you are correct! let me fix that

**Pull request merged**